src/main/java/com/programmers/domain/CalculatorType.java

```diff
@@ -0,0 +1,42 @@
+package com.programmers.domain;
+
+import java.util.Arrays;
+import java.util.function.BinaryOperator;
+
+public enum CalculatorType {
+    PLUS('+', (num1, num2) -> num1 + num2),
+    MINUS('-', (num1, num2) -> num1 - num2),
+    MULTIPLY('*', (num1, num2) -> num1 * num2),
+    DIVIDE('/', (num1, num2) -> {
+        isDividedByZero(num2);
+        return num1 / num2;
+    });
+
+    private static int ZERO_NUMBER = 0;
```

Comment:
```suggestion
    private static final int ZERO_NUMBER = 0;
```

static이 붙으면 전체 스레드에 공유를 하겠죠.
변경하지 않도록 final을 붙이는것이 중요합니다.

Reply:
네! 감사합니다ㅎㅎ 오늘 static이랑 jvm을 공부해봐야겠네요ㅎㅎ